app/models/peep.rb

```diff
@@ -0,0 +1,17 @@
+class Peep
+
```

Comment:
There needs to be an association with users here, you're adding the username, but what if the user changes their username?